clusterloader2/docs/test_suite_design.md

```diff
@@ -0,0 +1,41 @@
+# Cluster loader - TestSuite
+
+## Background
+
+Since 2019 cluster loader become main tool for kubernetes e2e performance tests.
+It is a very convenient tool for running single test case, but not for running
+test bundles. Currently it is impossible to run the same test
```

Comment:
nit:

It's very convenient for running single test case, but currently it's impossible to run the same test with two different sets of parameters in a single job run.

Reply:
Done.